corehq/apps/app_manager/templates/app_manager/partials/forms/case_config_ko_templates.html

```diff
@@ -284,11 +284,11 @@ <h4 class="panel-title panel-title-nolink">
           <!-- /ko -->
         </td>
         <td class="col-sm-2 text-center">
-          <div class="checkbox-save-only-if-edited">
-            <input type="checkbox"
-                   data-bind="click: $parent.switchSaveOnlyIfEdited, checked: $data.save_only_if_edited"
-                   value="$data.name"/>
-          </div>
+          <input type="checkbox"
+                 data-bind="click: $parent.switchSaveOnlyIfEdited,
+                            checked: $data.save_only_if_edited,
+                            visible: $parent.hasPrivilege"
```

Comment:
I'd disable this rather than hide it - the read-only version of the page shouldn't hide information.